service/src/main/java/edu/tamu/catalog/service/VoyagerCatalogService.java

```diff
@@ -280,4 +281,9 @@ public HoldingsRecord getHolding(String bibId, String holdingId) {
         }
         return null;
     }
+
+    @Override
+    public FeesFines getFeesFines(String uin) {
+        return null;
```

Comment:
Should we not throw an exception instead of returning null for these. https://commons.apache.org/proper/commons-lang/javadocs/api-3.4/org/apache/commons/lang3/NotImplementedException.html